app/src/main/java/pmediero/com/features/addplant/presentation/AddPlantScreen.kt

```diff
@@ -124,22 +129,34 @@ fun AddPlantScreen(
             }
 
         )
-        FooterAddPlantFigma(
+        FooterAddPlant(
             modifier = Modifier
                 .fillMaxWidth()
                 .padding(
                     horizontal = spacing.medium,
                     vertical = spacing.small
                 ),
             onFooterBtnClick = {
-                onEvent(OnCreatePlant)
+                onEvent(
+                    OnCreatePlantClick(
+                        Plant(
+                            name = state.plantName.asString(context),
```

Comment:
Estás utilizando el asString de fuera de Composables, tu quieres el que está anotado con Composable que no necesita el context.
`fun asString(context: Context): String {
        return when (this) {
            is DynamicString -> text
            is StringResource -> context.getString(resId)
        }
    }
    @Composable
    fun asString(): String {
        return when (this) {
            is DynamicString -> text
            is StringResource -> LocalContext.current.getString(resId)
        }
    }`